src/opcode/opcode.jl

```diff
@@ -112,74 +112,56 @@ include("./opcode_desc.jl")
 argument(op::OpCode) = argument(Val(op))
-argument(::Val{INT}) = Pickle.read_decimalnl_short
-argument(::Val{BININT}) = Pickle.read_int4
-argument(::Val{BININT1}) = Pickle.read_uint1
-argument(::Val{BININT2}) = Pickle.read_uint2
-argument(::Val{LONG}) = Pickle.read_decimalnl_long
-argument(::Val{LONG1}) = Pickle.read_long1
-argument(::Val{LONG4}) = Pickle.read_long4
-argument(::Val{STRING}) = Pickle.read_stringnl
-argument(::Val{BINSTRING}) = Pickle.read_string4
-argument(::Val{SHORT_BINSTRING}) = Pickle.read_string1
-argument(::Val{BINBYTES}) = Pickle.read_bytes4
-argument(::Val{SHORT_BINBYTES}) = Pickle.read_bytes1
-argument(::Val{BINBYTES8}) = Pickle.read_bytes8
-argument(::Val{BYTEARRAY8}) = Pickle.read_bytearray8
-argument(::Val{NEXT_BUFFER}) = nothing
-argument(::Val{READONLY_BUFFER}) = nothing
-argument(::Val{NONE}) = nothing
-argument(::Val{NEWTRUE}) = nothing
-argument(::Val{NEWFALSE}) = nothing
-argument(::Val{UNICODE}) = Pickle.read_unicodestringnl
-argument(::Val{SHORT_BINUNICODE}) = Pickle.read_unicodestring1
-argument(::Val{BINUNICODE}) = Pickle.read_unicodestring4
-argument(::Val{BINUNICODE8}) = Pickle.read_unicodestring8
-argument(::Val{FLOAT}) = Pickle.read_floatnl
-argument(::Val{BINFLOAT}) = Pickle.read_float8
-argument(::Val{EMPTY_LIST}) = nothing
-argument(::Val{APPEND}) = nothing
-argument(::Val{APPENDS}) = nothing
-argument(::Val{LIST}) = nothing
-argument(::Val{EMPTY_TUPLE}) = nothing
-argument(::Val{TUPLE}) = nothing
-argument(::Val{TUPLE1}) = nothing
-argument(::Val{TUPLE2}) = nothing
-argument(::Val{TUPLE3}) = nothing
-argument(::Val{EMPTY_DICT}) = nothing
-argument(::Val{DICT}) = nothing
-argument(::Val{SETITEM}) = nothing
-argument(::Val{SETITEMS}) = nothing
-argument(::Val{EMPTY_SET}) = nothing
-argument(::Val{ADDITEMS}) = nothing
-argument(::Val{FROZENSET}) = nothing
-argument(::Val{POP}) = nothing
-argument(::Val{DUP}) = nothing
-argument(::Val{MARK}) = nothing
-argument(::Val{POP_MARK}) = nothing
-argument(::Val{GET}) = Pickle.read_decimalnl_short
-argument(::Val{BINGET}) = Pickle.read_uint1
-argument(::Val{LONG_BINGET}) = Pickle.read_uint4
-argument(::Val{PUT}) = Pickle.read_decimalnl_short
-argument(::Val{BINPUT}) = Pickle.read_uint1
-argument(::Val{LONG_BINPUT}) = Pickle.read_uint4
-argument(::Val{MEMOIZE}) = nothing
-argument(::Val{EXT1}) = Pickle.read_uint1
-argument(::Val{EXT2}) = Pickle.read_uint2
-argument(::Val{EXT4}) = Pickle.read_int4
-argument(::Val{GLOBAL}) = Pickle.read_stringnl_noescape_pair
-argument(::Val{STACK_GLOBAL}) = nothing
-argument(::Val{REDUCE}) = nothing
-argument(::Val{BUILD}) = nothing
-argument(::Val{INST}) = Pickle.read_stringnl_noescape_pair
-argument(::Val{OBJ}) = nothing
-argument(::Val{NEWOBJ}) = nothing
-argument(::Val{NEWOBJ_EX}) = nothing
-argument(::Val{PROTO}) = Pickle.read_uint1
-argument(::Val{STOP}) = nothing
-argument(::Val{FRAME}) = Pickle.read_uint8
-argument(::Val{PERSID}) = Pickle.read_stringnl_noescape
-argument(::Val{BINPERSID}) = nothing
+
+
+for (ops, f) in (
```

Comment:
It is better to isolate the series of ops and fs as an object, then you put it after for loop. For example, `OPS_MAP`.

Reply:
I don't get what you mean. What's the purpose of that `OPS_MAP`?